.github/workflows/auto-cherry-picker.yaml

```diff
@@ -14,23 +14,88 @@ on:
         required: true
 
 jobs:
-  picker:
+  prerequisites:
+    name: Gather Prerequisites
     if: (github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'needs-cherrypick') ) || github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
+    outputs:
+      pr_num: ${{ steps.get-prereqs.outputs.pr_num }}
+      merge_commit: ${{ steps.get-prereqs.outputs.merge_commit }}
+      matrix: ${{ steps.get-prereqs.outputs.matrix }}
     steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
The current node LTS seems to be 18, did you choose 16 for a particular reason?

Reply:
Copy pasta from the actions script readme